include/simdjson/compiler_check.h

```diff
@@ -56,10 +56,21 @@
 #endif
 #endif
 
-#ifdef __cpp_concepts
+#ifdef __clang__
+// concepts are not supported in LLVM 12, see
+// https://libcxx.llvm.org/Status/Cxx20.html
+#if __clang_major__ < 13
```

Comment:
this is an extremely old clang version. i recommend keeping as it is, since nodejs infra will update soon.

Reply:
Ok. Removing...

Reply:
I will leave SIMDJSON_CONCEPT_DISABLED for now so that it can be disabled manually.